doc/source/cluster/vms/user-guides/community/slurm.rst

```diff
@@ -107,46 +101,27 @@ Next, we'll want to obtain a hostname and a node IP address for the head node. T
    :start-after: __doc_head_address_start__
    :end-before: __doc_head_address_end__
 
+.. note:: In Ray 2.49 and above, you can use IPv6 addresses/hostnames.
 
 
-Starting the Ray head node
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+Starting Ray and executing your script
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note:: `ray symmetric-run` is available in Ray 2.49 and above. Check older versions of the documentation if you are using a older version of Ray.
```

Comment:
consider a call to action if people are using `symmetric-run` and run into limitations/issues?